docs/source/guides/extensions.rst

```diff
@@ -170,8 +170,8 @@ This extension ``PolynomialShift`` takes five arguments.
 
 - ``attr``: The name of the optimizer property you want to update using this extension.
 - ``power``: The power of the above equation to calculate the learning rate.
-- ``stop_trigger``: The trigger given to the :class:`~chainer.traininig.Trainer` object to specify when to stop the training loop.
+- ``stop_trigger``: The trigger given to the :class:`~chainer.training.Trainer` object to specify when to stop the training loop.
 - ``batchsize``: The training mini-batchsize.
 - ``len_dataset``: The length of the dataset, i.e., the number of data in the training dataset.
 
-This extension calculates the number of iterations which will be performed during training by using ``stop_trigger``, ``batchsize``, and ``len_dataset``, then stores it as a property ``_maxiter``. This property will be used in the :meth:`__call__` method to update the learning rate. The :meth:`initialize` method obtains the initial learning rate from the optimizer given to the :class:`~chainer.training.Trainer` object. The :meth:`~chainer.traininig.serialize` method stores or recovers the properties, ``_t`` (number of iterations) and ``_last_value`` (the latest learning rate), belonging to this extension.
+This extension calculates the number of iterations which will be performed during training by using ``stop_trigger``, ``batchsize``, and ``len_dataset``, then stores it as a property ``_maxiter``. This property will be used in the :meth:`~chainer.training.extensions.PolynomialShift.__call__` method to update the learning rate. The :meth:`~chainer.training.extensions.PolynomialShift.initialize` method obtains the initial learning rate from the optimizer given to the :class:`~chainer.training.Trainer` object. The :meth:`~chainer.training.extensions.PolynomialShift.serialize` method stores or recovers the properties, ``_t`` (number of iterations) and ``_last_value`` (the latest learning rate), belonging to this extension.
```

Comment:
`chainer.training.extensions.PolynomialShift` is not appropriate, because even with the same name, the one in the example is different from it.